### PR TITLE
GG-27348 Implemented support for Ent/Ult agent.

### PR DESCRIPTION
**Pull request by akuznetsov-gridgain**

### CODE REVIEW

File: modules/management-console-agent/src/main/java/org/apache/ignite/agent/action/controller/SecurityActionsController.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
     public SecurityActionsController(GridKernalContext ctx) {
         this.ctx = ctx;
         this.log = ctx.log(SecurityActionsController.class);
-        this.registry = ((ManagementConsoleProcessor)ctx.managementConsole()).sessionRegistry();
+        this.registry = ((ManagementConsoleAgent)ctx.managementConsole()).sessionRegistry();
```
Comment (nva) [inline modules/management-console-agent/src/main/java/org/apache/ignite/agent/action/controller/SecurityActionsController.java]: Why we need this cast. Can we add this method to interface?

File: modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/action/DistributedActionProcessor.java [hunk 2]
```diff
@@ -61,9 +61,9 @@
     public DistributedActionProcessor(GridKernalContext ctx) {
         super(ctx);
 
-        this.mgr = ((ManagementConsoleProcessor)ctx.managementConsole()).webSocketManager();
-        this.sesRegistry = ((ManagementConsoleProcessor)ctx.managementConsole()).sessionRegistry();
-        this.locNodeConsistentId = String.valueOf(ctx.grid().localNode().consistentId());
+        mgr = ((ManagementConsoleAgent)ctx.managementConsole()).webSocketManager();
```
Comment (nva) [inline modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/action/DistributedActionProcessor.java]: Why we need this cast. Can we add this method to interface?

File: modules/management-console-agent/src/main/java/org/apache/ignite/agent/action/controller/SecurityActionsController.java (hunk 1 above)
Comment (akuznetsov-gridgain) [reply modules/management-console-agent/src/main/java/org/apache/ignite/agent/action/controller/SecurityActionsController.java]: No, this is implementation details so far.

File: modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/action/DistributedActionProcessor.java (hunk 2 above)
Comment (akuznetsov-gridgain) [reply modules/management-console-agent/src/main/java/org/apache/ignite/agent/processor/action/DistributedActionProcessor.java]: No, this is implementation details so far.

**Pull request merged**